alerta-covid/db/migrate/20210326062347_create_areas.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAreas < ActiveRecord::Migration[6.1]
+  def change
+    create_table :areas do |t|
+      t.references :institution, null: false, foreign_key: true
+      t.string :name
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Is a good practice to keep migrations in a single commit. But it's not required

Reply:
Thanks for the observation I will take it into account in the following commits